### PR TITLE
*/*: various QA fixes

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: app-misc/gcstar/gcstar-1.7.1-r1.ebuild [hunk 1]
```diff
@@ -65,15 +65,3 @@ src_install() {
 
 	dodoc CHANGELOG README README.fr
 }
-
-pkg_postinst() {
```
Comment (SoapGentoo) [inline app-misc/gcstar/gcstar-1.7.1-r1.ebuild:69]: :+1:

File: app-misc/xmind/xmind-3.5.3.201506180105-r2.ebuild [hunk 2]
```diff
@@ -81,9 +81,11 @@ pkg_preinst() {
 
 pkg_postinst() {
 	gnome2_icon_cache_update
```
Comment (SoapGentoo) [inline app-misc/xmind/xmind-3.5.3.201506180105-r2.ebuild:83]: try completely obliterating `gnome2-utils`

File: app-misc/xmind/xmind-3.7.6_p201801311814.ebuild [hunk 3]
```diff
@@ -85,9 +85,11 @@ pkg_preinst() {
 pkg_postinst() {
 	gnome2_icon_cache_update
```
Comment (SoapGentoo) [inline app-misc/xmind/xmind-3.7.6_p201801311814.ebuild:86]: same

File: media-sound/terminatorx/terminatorx-4.0.1-r2.ebuild [hunk 4]
```diff
@@ -49,6 +49,12 @@ PATCHES=(
 	"${FILESDIR}"/${P}-gtkcombotext.patch
 )
 
+src_prepare() {
+	# 740502
+	sed -i 's/\Application;//g' terminatorX.desktop  || die "Sed failed"
```
Comment (SoapGentoo) [inline media-sound/terminatorx/terminatorx-4.0.1-r2.ebuild:54]: turn this into a patch

File: media-video/parole/parole-1.0.5.ebuild [hunk 5]
```diff
@@ -37,6 +37,12 @@ DEPEND="${COMMON_DEPEND}
 
 DOCS=( AUTHORS ChangeLog README THANKS TODO )
 
+src_prepare() {
+	# 740516
+	sed '/OnlyShowIn/d' data/desktop/desktop.sh || die "Sed failed"
```
Comment (SoapGentoo) [inline media-video/parole/parole-1.0.5.ebuild:42]: turn into a patch (regardless, are you sure you aren't missing `-i`?)

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-misc/xmind/xmind-3.7.6_p201801311814.ebuild [hunk 6]
```diff
@@ -78,16 +78,9 @@ src_install() {
...
-pkg_preinst() {
-	gnome2_icon_savelist
-}
-
 pkg_postinst() {
-	gnome2_icon_cache_update
 	font_pkg_postinst
-	elog "For audio notes support, install media-sound/lame"
+	xdg_pkg_postinst
+	optfeature "For audio notes support, install media-sound/lame"
```
Comment (SoapGentoo) [inline app-misc/xmind/xmind-3.7.6_p201801311814.ebuild:84]: `optfeature "audio notes support" media-sound/lame`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-misc/xmind/xmind-3.5.3.201506180105-r2.ebuild [hunk 7]
```diff
@@ -75,15 +74,8 @@ src_install() {
...
-pkg_preinst() {
-	gnome2_icon_savelist
-}
-
 pkg_postinst() {
-	gnome2_icon_cache_update
-	elog "For audio notes support, install media-sound/lame"
+	xdg_pkg_postinst
+	optfeature "audio notes support" media-sound/lame
 }
```
Comment (SoapGentoo) [inline app-misc/xmind/xmind-3.5.3.201506180105-r2.ebuild:81]: trailing empty line ;)

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-misc/xmind/xmind-3.5.3.201506180105-r2.ebuild (hunk 7 above)
Comment (jsmolic) [reply app-misc/xmind/xmind-3.5.3.201506180105-r2.ebuild:81]: @SoapGentoo nice catch :)

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**